tests/framework/installer/ceph_helm_installer.go

```diff
@@ -63,10 +78,173 @@ func (h *CephInstaller) CreateRookCephClusterViaHelm(values map[string]interface
 	}
 	values["cephClusterSpec"] = clusterCRD["spec"]
 
+	if err := h.CreateBlockPoolConfiguration(values, blockPoolName, blockPoolSCName); err != nil {
+		return err
+	}
+	if err := h.CreateFileSystemConfiguration(values, filesystemName, filesystemSCName); err != nil {
+		return err
+	}
+	if err := h.CreateObjectStoreConfiguration(values, objectStoreName, objectStoreSCName); err != nil {
+		return err
+	}
+
 	logger.Infof("Creating ceph cluster using Helm with values: %+v", values)
 	if err := h.helmHelper.InstallLocalRookHelmChart(h.settings.Namespace, CephClusterChartName, values); err != nil {
 		return err
 	}
 
+	logger.Infof("Confirming ceph cluster installed correctly")
+	if err := h.confirmHelmClusterInstalledCorrectly(); err != nil {
```

Comment:
Instead of confirming this immediately, how about calling it just after the call to [WaitForToolbox()](https://github.com/rook/rook/blob/643eddf261e9f59ff9f5e9312d33e9369c5ef958/tests/framework/installer/ceph_installer.go#L530)? This way, the standard cluster validations will be done first, and then these new validations.

Reply:
done, also added the cleanup in the case of `RetainHelmDefaultStorageCRs` ever being used in the future